Marlin/src/lcd/dogm/status_screen_DOGM.cpp

```diff
@@ -362,7 +362,7 @@ void MarlinUI::draw_status_screen() {
   #endif
 
   #if DO_DRAW_CHAMBER
-    #if HAS_HEATED_CHAMBER && defined(STATUS_CHAMBER_ANIM)
+    #if HAS_HEATED_CHAMBER && ENABLED(STATUS_CHAMBER_ANIM)
```

Comment:
Following your `STATUS_CHAMBER_ANIM` undef when no heated chamber, this may be `#if ENABLED(STATUS_CHAMBER_ANIM)`, I think